rootfs/conf/kops/helmfile.d/0620.portal.yaml

```diff
@@ -34,218 +34,139 @@ releases:
-
-    - name: "oauth2-proxy.app.githubOrg"
-      value: '{{ env "PORTAL_OAUTH2_PROXY_GITHUB_ORGANIZATION" }}'
-
-    - name: "oauth2-proxy.app.githubTeam"
-      value: '{{ env "PORTAL_OAUTH2_PROXY_GITHUB_TEAM" }}'
-
-    - name: "oauth2-proxy.app.requestLogging"
-      value: "false"
-
-    - name: "oauth2-proxy.app.passAccessToken"
-      value: "false"
-
-    - name: "oauth2-proxy.app.passBasicAuth"
-      value: "false"
-
-    - name: "oauth2-proxy.app.passHostHeader"
-      value: "true"
-
-    - name: "oauth2-proxy.app.cookieName"
-      value: '{{ env "PORTAL_OAUTH2_PROXY_COOKIE_NAME" }}'
-
-    - name: "oauth2-proxy.app.cookieSecret"
-      value: '{{ env "PORTAL_OAUTH2_PROXY_COOKIE_SECRET" }}'
-
-    - name: "oauth2-proxy.app.cookieDomain"
-      value: '.{{ env "PORTAL_HOSTNAME" }}'
-
-    - name: "oauth2-proxy.app.cookieSecure"
-      value: "true"
-
-    - name: "oauth2-proxy.app.cookieHttponly"
-      value: "false"
-
-    - name: "oauth2-proxy.app.replicaCount"
-      value: '{{ env "PORTAL_OAUTH2_PROXY_REPLICA_COUNT" | default "2" }}'
-
-    - name: "oauth2-proxy.app.image.repository"
-      value: "cloudposse/oauth2-proxy"
-
-    - name: "oauth2-proxy.app.image.tag"
-      value: '{{ env "PORTAL_OAUTH2_PROXY_IMAGE_TAG" | default "2.2" }}'
-
-    - name: "oauth2-proxy.app.image.pullPolicy"
-      value: "Always"
-
-    - name: "oauth2-proxy.app.resources.limits.cpu"
-      value: "10m"
-
-    - name: "oauth2-proxy.app.resources.limits.memory"
-      value: "16Mi"
-
-    - name: "oauth2-proxy.app.resources.requests.cpu"
-      value: "5m"
-
-    - name: "oauth2-proxy.app.resources.requests.memory"
-      value: "8Mi"
-
-    ## Ingress
-    - name: "ingress.enabled"
-      value: "true"
-
-    - name: "ingress.dns.enabled"
-      value: "false"
-
-    - name: "ingress.tls.enabled"
-      value: "true"
-
-    - name: "ingress.annotations.kubernetes\\.io/ingress\\.class"
-      value: "nginx"
-
-    # FIXME: this is blocked by https://github.com/roboll/helmfile/issues/119
-    #- name: "ingress.annotations.kubernetes\\.io/tls-acme"
-    #  value: "true"
-
-    ### Required: PORTAL_INGRESS; e.g. ingress.us-west-2.staging.cloudposse.org
-    - name: "ingress.annotations.external-dns\\.alpha\\.kubernetes\\.io/target"
-      value: '{{ env "PORTAL_INGRESS" }}'
-
-    # DNS TTL for PORTAL_HOSTNAME; e.g. 60
-    # FIXME: this is blocked by https://github.com/roboll/helmfile/issues/119
-    #- name: "ingress.annotations.external-dns\\.alpha\\.kubernetes\\.io/ttl"
-    #  value: "60"
+    - backends:
+        dashboard:
+          ### Optional: PORTAL_BACKEND_K8S_DASHBOARD_NAME;
+          name: '{{ env "PORTAL_BACKEND_K8S_DASHBOARD_NAME" | default "Kubernetes" }}'
+          ### Optional: PORTAL_BACKEND_K8S_DASHBOARD_ENDPOINT;
+          endpoint: '{{ env "PORTAL_BACKEND_K8S_DASHBOARD_ENDPOINT" | default "https://kubernetes-dashboard.kube-system:443" }}'
+          external: "false"
```

Comment:
I tested this and it doesn't work. Needs to be literal boolean `false` without quotes.